src/Type/Definition/FieldDefinition.php

```diff
@@ -7,7 +7,9 @@
 use GraphQL\Error\Error;
 use GraphQL\Error\InvariantViolation;
 use GraphQL\Language\AST\FieldDefinitionNode;
+use GraphQL\Type\Schema;
 use GraphQL\Utils\Utils;
+use function assert;
```

Comment:
Might be unused. Wonder why it was not caught by CS check 🤔

Reply:
f89c842